hoss/hoss_config.json

```diff
@@ -59,24 +59,6 @@
       "Epoch": "2018-01-01T00:00:00.000000"
     }
   ],
-  "Grid_Mapping_Data": [
-    {
-      "Grid_Mapping_Dataset_Name": "EASE2_Global",
-      "grid_mapping_name": "lambert_cylindrical_equal_area",
-      "standard_parallel": 30.0,
-      "longitude_of_central_meridian": 0.0,
-      "false_easting": 0.0,
-      "false_northing": 0.0
-    },
-    {
-      "Grid_Mapping_Dataset_Name": "EASE2_Polar",
-      "grid_mapping_name": "lambert_azimuthal_equal_area",
-      "longitude_of_projection_origin": 0.0,
-      "latitude_of_projection_origin": 90.0,
-      "false_easting": 0.0,
-      "false_northing": 0.0
-    }
-  ],
```

Comment:
I'm not hard over on this, but I had thought the grid-mapping variables would be handled slightly differently, and I guess I didn't catch this in the VarInfo review.  I thought the applicability reference would be to a "phantom" variable reference (name fully enumerated, path might be pattern match or fully enumerated). This declaration might have to exist (be declared) following rules that establish a reference to this variable, or the order might not matter.  In any case, the phantom variable would have to exist as an attribute reference value from one or more science variables. VarInfo would be able to create attributes on this phantom variable, and the application (HOSS) could ask for the attributes for this variable, even if the variable itself did not exist in VarInfo's variable list.  This might actually be the VarInfo 3.0 changes, so moot at this point, but I would want to check on that.

Reply:
Will be done as part of DAS-2256. updated the ticket with the above information

Reply:
@autydp said it can be marked resolved.